main.js

```diff
@@ -0,0 +1,104 @@
+// create eventlistener and variable
+
+document.getElementById('button-search').addEventListener('click', function() {
+  let townName = document.getElementById('current-town').value;
+  
+  weatherOfTheDay(townName);
+
+});
+
+//create new function 
+
+function weatherOfTheDay(townName) {
+
+      const apiKey = '63168e96197ab571649bdefbef398926';
+      const currentWeatherApiUrl = `https://api.openweathermap.org/data/2.5/weather?q=${townName}&units=imperial&appid=${apiKey}`;
+//create fetch to grab data, add error check as well
+    fetch(currentWeatherApiUrl)
+            .then(response => response.json())
+            .then(data => {
```

Comment:
be more descriptive, data tells not much, 
```suggestion
            .then(weatherData => {
```